dot/parachain/bitfield-distribution/bitfield_distribution.go

```diff
@@ -470,16 +491,48 @@ func (b *BitfieldDistribution) processIncomingPeerMessageEvent(event networkbrid
 }
 
 func (b *BitfieldDistribution) processUpdatedAuthorityIDsEvent(event networkbridgeevents.UpdatedAuthorityIDs) error {
-	//TODO implement in #4360
-	panic("implement me")
+	logger.Tracef("process UpdatedAuthorityIDs event: %v", event)
+
+	ids := make(map[types.AuthorityID]struct{})
+	for _, id := range event.AuthorityDiscoveryIDs {
+		ids[types.AuthorityID(id)] = struct{}{}
+	}
+	ok, err := b.topologies.CurrentTopology.UpdateAuthoritiesIDs(event.PeerID, ids)
+	if err != nil {
+		return err
+	}
+	if !ok {
+		logger.Errorf("could not update authority IDs : %v", event.AuthorityDiscoveryIDs)
+		return nil
+	}
+
+	return nil
```

Comment:
is there a reason for `ok` be false? Also the `errorf` log should be a `warnf` given that it is not an error but an unexpected result

```suggestion
	if err != nil {
	        logger.Errorf("error while updating authority IDs : %s", err.Error())
		return err
	}
	
	if !ok {
	        logger.Warnf("could not update authority IDs : %s", err.Error())
	}

	return nil
```

Reply:
According to `UpdateAuthoritiesIDs()`, if the given peer already exists in the CurrentTopology, it will not update so that it will return false without error.